platform/vcpkg/triplets/arm64-ioscross.cmake

```diff
@@ -0,0 +1,15 @@
+set(VCPKG_TARGET_ARCHITECTURE arm64)
+set(VCPKG_CRT_LINKAGE dynamic)
+set(VCPKG_LIBRARY_LINKAGE static)
+
+set(VCPKG_CMAKE_SYSTEM_NAME iOS)
+set(VCPKG_OSX_ARCHITECTURES arm64)
+
+set(CMAKE_OSX_SYSROOT $ENV{OSXCROSS_SDK})
+
+set(VCPKG_C_FLAGS "-I${CMAKE_OSX_SYSROOT}/usr/include")
+set(VCPKG_CXX_FLAGS "-I${CMAKE_OSX_SYSROOT}/usr/include -stdlib=libc++")
+set(VCPKG_LINKER_FLAGS "-L${CMAKE_OSX_SYSROOT}/usr/lib")
+
+get_filename_component(TOOLCHAIN_FILE "../platform/ioscross/ios.cmake" REALPATH BASE_DIR "${CMAKE_SOURCE_DIR}")
+set(VCPKG_CHAINLOAD_TOOLCHAIN_FILE ${TOOLCHAIN_FILE})

```

Comment:
Afaik this should be in the `CMakeLists.txt` and not this triplet file

Reply:
I tried removing this file but when using a toolchain vcpkg ignores most of the vcpkg flags and relies on the toolchain to set the values.
The vckpg [documentation](https://learn.microsoft.com/en-us/vcpkg/users/triplets#vcpkg_chainload_toolchain_file) it specifies `VCPKG_CHAINLOAD_TOOLCHAIN_FILE` being an option for triplets.  Not sure how to go about cross compiling without specifying this value.